modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -28,6 +28,120 @@ def self.date_from_string(string)
       if: proc { |a| a.form_data.present? }
     )
 
+    def pdf_structure(version)
```

Comment:
I would add this method together with the `Structure` object

Reply:
Our `PdfConstruction::Generator` model calls `pdf_structure` on the appeal, in order to get a reference to the Structure instance. However, in this specific case, there's no PdfConstruction::SupplementalClaim::Structure` in our code base. I'm saying that logically, you should add the reference to the structure with the pdf_structure interface in the same Pull Request. Does that make sense?